### ISSUE TITLE
[chain-id] specifying Rooch Chain ID on MetaMask doesn't work

### ISSUE THREAD

**Issue by feliciss** [rooch-network/rooch#690]
Introduced by PR #681 @jolestar 

Entered same chain id as in `chain_id.rs` but got errored.

Screenshot:

![err cleaned](https://github.com/rooch-network/rooch/assets/22887031/a284b778-a576-4b02-bb03-367684c3e9b7)

Update:

Beginning with `0x` works:

![err-update cleaned](https://github.com/rooch-network/rooch/assets/22887031/038e111b-16ea-46a6-9901-a204235f98b5)

**Comment by jolestar** [comment]
The ethereum chain_id:

https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_chainid

```
// Request
curl -X POST --data '{"jsonrpc":"2.0","method":"eth_chainId","params":[],"id":67}'
// Result
{
  "id":67,
  "jsonrpc": "2.0",
  "result": "0x1"
}
```

rooch RPC response:
```bash
curl -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"eth_chainId","params":[],"id":67}' http://localhost:50051
```
```json
{"jsonrpc":"2.0","result":"0x134AFD7","id":67}
```

I do not know why it did not work.

@yubing744

**Comment by jolestar** [comment]
The hex need to be lowercase, fixed in #693
